server/test/testing-constants.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * a simple file for mostly holding testing constants
+ */
+
+import { TypeOrmModuleOptions } from '@nestjs/typeorm';
+
+export const TEST_DB_OPTIONS: TypeOrmModuleOptions = {
+  name: 'default',
+  type: 'postgres',
+  host: 'localhost',
+  port: 5432,
+  username: 'postgres',
+  password: 'your_password',
+  database: 'e2e',
```

Comment:
would be ideal if we could use `${process.env.DATABASE_DB}` and other .env vars here